django/contrib/admindocs/views.py

```diff
@@ -38,7 +38,7 @@ def dispatch(self, request, *args, **kwargs):
         return super(BaseAdminDocsView, self).dispatch(request, *args, **kwargs)
 
     def get_context_data(self, **kwargs):
-        kwargs.update({'root_path': urlresolvers.reverse('admin:index')})
+        kwargs.update({'root_path': urls.reverse('admin:index')})
```

Comment:
What do you think about standardizing the usage to import each function individually? (here and elsewhere)